### ISSUE TITLE
Master: Invisible Text in Parts over Multimeasure rests completely vanish

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#25598]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. Make a part with text over a multimeasure rest
2. In the part, make the text invisible, it completely disappears

After selecting "Reset All Formatting" in the instruments panel on the part, the invisible text is not restored

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/8e1f5dbb-5a22-4b8a-bc9b-7ea5c2449b29

### What is the latest version of MuseScore Studio where this issue is present?

OS: macOS 14.6, Arch.: arm64, MuseScore Studio version (64-bit): 4.5.0-243260309, revision: github-musescore-musescore-9fd3271

### Regression

Yes, this works in the latest released version and is broken in the nightly builds

### Operating system

Mac OS 14

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by zacjansheski** [comment]
They can be accessed by turning off multimeasure rests, so perhaps this is not an issue at all

**Comment by mike-spa** [comment]
I think that's expected behaviour. Those items are visible because their existance interrupts the multirests. But invisible elements do not break multirests (and they shouldn't), therefore once you make them invisible they become "absorbed" in the multirest. @oktophonie am I missing something?

**Comment by zacjansheski** [comment]
@mike-spa makes sense to me, but I do think they need to be restored after "reset all formatting" (Sorry that wasn't really in the description)
